src/renderer/components/+config/config.route.ts

```diff
@@ -1,12 +1,21 @@
 import { RouteProps } from "react-router";
-import { Config } from "./config";
 import { IURLParams } from "../../../common/utils/buildUrl";
-import { configMapsURL } from "../+config-maps/config-maps.route";
+import { configMapsRoute, configMapsURL } from "../+config-maps/config-maps.route";
+import { hpaRoute } from "../+config-autoscalers";
+import { limitRangesRoute } from "../+config-limit-ranges";
+import { pdbRoute } from "../+config-pod-disruption-budgets";
+import { resourceQuotaRoute } from "../+config-resource-quotas";
+import { secretsRoute } from "../+config-secrets";
 
 export const configRoute: RouteProps = {
-  get path() {
-    return Config.tabRoutes.map(({ routePath }) => routePath).flat();
-  }
+  path: [
+    configMapsRoute,
+    secretsRoute,
+    resourceQuotaRoute,
+    limitRangesRoute,
+    hpaRoute,
+    pdbRoute
+  ].map(route => route.path.toString())
```

Comment:
```suggestion
  path: [
    configMapsRoute.path,
    secretsRoute.path,
    resourceQuotaRoute.path,
    limitRangesRoute.path,
    hpaRoute.path,
    pdbRoute.path,
  ].map(String)
```

Reply:
Not sure what option is better. Are we okay with lots of `.path` repeats?